docs/index.md

```diff
@@ -101,6 +101,7 @@ device state, player variables, etc.
   * [Shows](shows/index.md)
   * [MPF Errors from Log Files](logs/index.md)
   * [Example Games](examples/index.md)
+  * [Code Extensions](code/index.md)
```

Comment:
This is a great addition. For consistency, I think we should refer to "custom code" as a term, so maybe "Extending MPF with Custom Code" or something like that?

Reply:
changed.